powerfulseal/policy/action_nodes_pods.py

```diff
@@ -208,6 +209,7 @@ def action_wait(self, items, params):
         sleep_time = params.get("seconds", 0)
         self.logger.info("Action sleep for %s seconds", sleep_time)
         time.sleep(sleep_time)
+        return True
```

Comment:
curious what does `action_wait` need a return value?

Reply:
Every action needs a return value to signal whether it succeeded or not.